interp/src/primitives/stateful.rs

```diff
@@ -538,6 +448,7 @@ impl StdMemD1 {
             idx_size, //the width of the values used to address the memory
             data,
             update: None,
+            cycle_count: 0,
```

Comment:
Same comments about storing inputs and the cycle_count variable from the register